noggin/tests/unit/controller/test_user_otp.py

```diff
@@ -12,17 +12,19 @@
     assert_form_field_error,
     assert_form_generic_error,
     assert_redirects_with_flash,
+    get_otp,
+    otp_secret_from_uri,
 )
 
 
 @pytest.fixture
-def dummy_user_with_2_otp(client, logged_in_dummy_user, dummy_user_with_otp):
+def dummy_user_with_2_otp(client, logged_in_dummy_user, logged_in_dummy_user_with_otp):
```

Comment:
Same remark here.

Reply:
I think here it would be needed because `logged_in_dummy_user_with_otp` only returns the otp token parsed and a few lines below `logged_in_dummy_user` is used to add a new token. Maybe this fixture would be added in `conftest.py` or `logged_in_dummy_user_with_otp` would be improved later to returns the client handler with the token.

Reply:
Correct, I missed that.

I agree that ideally `logged_in_dummy_user_with_otp` would return both the handler and the token. It'll be quite a few changes throughout the tests code, so it's great if you're willing to do that, but if you don't I won't hold this PR for that.